R/survival-helper.R

```diff
@@ -180,6 +180,85 @@ km_plot <- function(km_fit_before, km_fit_after = NULL, time_scale, trt, trt_ext
 }
 
 
+#' Function to plot Kaplan-Meier curves using survminer package
+#'
+#' @param combined_data combined data of internal IPD and psuedo IPD from aggregate data.
+#' @param trt internal trial treatment
+#' @param trt_ext external trial treatment
+#' @param trt_common treatment that is shared between internal and external trial. Only applies in anchored comparison.
+#' @param break.x.by bin parameter for survminer
+#' @param endpoint_name a character name of the endpoint
+#' @param censor indicator to include censor information
+#' @param risk.table indicator to include risk table
```

Comment:
do you prefer to explicitly have these parameters or just pass them through as `...`?